hack/tools/go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/google/go-flow-levee v0.1.5
 	gotest.tools/gotestsum v1.6.4
 	honnef.co/go/tools v0.3.3
-	sigs.k8s.io/logtools v0.1.0
+	sigs.k8s.io/logtools v0.2.1-0.20220830112100-f68289b649a5
```

Comment:
Let's do a formal release of logtools first. For that I am waiting for https://github.com/kubernetes-sigs/logtools/pull/1